src/tests/Cell/Cell.test.tsx

```diff
@@ -1,14 +1,7 @@
-const mockedUsedNavigate = jest.fn();
-jest.mock('react-router-dom', () => ({
-   ...jest.requireActual('react-router-dom'),
-  useNavigate: () => mockedUsedNavigate,
-}));
 /*
-ToDo: обращение к проверяющему:
-    Можно ли как то замокать useNavigate один раз для всех тестов ?
-    Так же как замокать для storybook, из за useNavigate, половина компонент
-    в нем не работает теперь:
-    "usenavigate() may be used only in the context of a <router> component."
+    ToDo: поломался storybook, из за useNavigate, половина компонент
+        в нем не работает теперь:
+        "usenavigate() may be used only in the context of a <router> component."
```

Comment:
https://stackoverflow.com/questions/70029935/react-router-v6-usenavigate-may-be-used-only-in-the-context-of-a-router-co